### PR TITLE
chore: Update to generator v1.4.9

### PR DESCRIPTION
**Pull request by jskeet**
The new generator uses .NET 6.0 and doesn't generate unit tests.

The changes in this commit are mostly to remove post-generation fix-ups for unit tests, now that we don't have the unit tests.

### CODE REVIEW

**Comment by jskeet** [comment]
This mustn't be merged until we release the new generator version and update to it, but then it's required in order to use that correctly.

**Comment by jskeet** [comment]
Oooh... we'll need to remove all the test projects from solutions as well. Will work on that...

**Comment by jskeet** [comment]
(And will need a version of GAX that supports WithHttpRuleOverrides... so we'll need to do a GAX release too.)

File: generateapis.sh [hunk 1]
```diff
@@ -54,6 +54,8 @@ generate_microgenerator() {
   # Delete previously-generated files
   delete_generated apis/$1/$1
   delete_generated apis/$1/$1.Tests
+  rm -f apis/$1/$1.Tests/$1.Tests.csproj
+  rm -f apis/$1/$1.Tests/coverage.xml
```
Comment (amanda-tarafa) [inline generateapis.sh:58]: These are a one off, righ?

**Review by amanda-tarafa — APPROVED** [review]
LGTM

**Comment by jskeet** [comment]
I think the easiest way to remove the test projects will be to remove *all* solution files under apis, then use generate-projects.sh to regenerate them from scratch :)

**Comment by jskeet** [comment]
(Will rebase to handle the conflict.)

**Review by amanda-tarafa — APPROVED** [review]
LGTM

**Pull request merged**